.github/workflows/deploy-prod.yaml

```diff
@@ -88,7 +88,7 @@ jobs:
       - name: Upload inframap
         id: inframap
         run: |
-          IMGBBURL=$(curl -s -L -X POST "https://api.imgbb.com/1/upload?key=${{ secrets.IMGBB_API_TOKEN }}" --form "image=@/tmp/inframap.png" | jq -r '.data.url')
+          IMGBBURL=$(curl -s -L -X POST "https://api.imgbb.com/1/upload?key=${{ vars.IMGBB_API_TOKEN }}" --form "image=@/tmp/inframap.png" | jq -r '.data.url')
```

Comment:
This should still be a secret

Reply:
We could probably remove the whole infra map feature. It was a cool demo but I think it just adds unnecessary complexity now.